RadarSDK/RadarAPIClient.m

```diff
@@ -251,10 +251,12 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
         params[@"floorLevel"] = @(location.floor.level);
     }
     long nowMs = (long)([NSDate date].timeIntervalSince1970 * 1000);
-    if (!foreground) {
-        long timeInMs = (long)(location.timestamp.timeIntervalSince1970 * 1000);
-        params[@"updatedAtMsDiff"] = @(nowMs - timeInMs);
+    long locationMs = (long)(location.timestamp.timeIntervalSince1970 * 1000);
+    RadarTrackingOptions *options = [Radar getTrackingOptions];
+    if (!verified || !foreground) {
```

Comment:
so verified background reqs will use `updatedAtMsDiff`, is that intended?
Are we intending to change how we treat foreground timestamps or holding off on that bc it's a bigger change not needed for the fix?  

Would recommend updating the PR description with a high level description of the changes to make it easier to understand.

Reply:
Yes that's correct. Per slack conversation with Nick, the intention is to not change any timestamp logic for `verified` requests.

All non verified `/tracks` will now use `updatedAtMsDiff`, both foreground and background. There's no reason to hold off on making (non-verified) foreground requests use this logic -- in fact it's required for making sure that foreground requests slot in correctly in between background requests.

Ooo yea, good call out re: pr description -- just updated it.

Reply:
should the code be:
```
if (!verified) {
```

as is it doesn't match your comment.  Unless I'm missing something

Reply:
We do want background verified tracks to use the existing logic, i.e. `if (!foreground) ...`

So either `if (!verified || !foreground) {`

Does that make sense?